stock_picking_return_restricted_qty/wizard/stock_picking_return.py

```diff
@@ -17,12 +17,17 @@ def _prepare_stock_return_picking_line_vals_from_move(self, stock_move):
         return val
 
     def _create_returns(self):
+        restrict_return_qty = self.picking_id.picking_type_id.restrict_return_qty
```

Comment:
You did it well. In fact, just the parameter on picking type level should be sufficient.

Could you adapt CONFIGURATION fragment to reflect this change ? Thanks